Mathlib/Analysis/InnerProductSpace/Calculus.lean

```diff
@@ -245,6 +245,40 @@ theorem DifferentiableAt.norm (hf : DifferentiableAt ℝ f x) (h0 : f x ≠ 0) :
   ((contDiffAt_norm 𝕜 h0).differentiableAt le_rfl).comp x hf
 #align differentiable_at.norm DifferentiableAt.norm
 
+theorem not_differentiableAt_abs_zero : ¬ DifferentiableAt ℝ (fun x : ℝ ↦ abs x) 0 := by {
```

Comment:
```suggestion
theorem not_differentiableAt_abs_zero : ¬ DifferentiableAt ℝ (fun x : ℝ ↦ abs x) 0 := by
```
We don't actually use the braces much these days (they show up in the error message though which is a bit annoying as it makes it seem like they are needed, they aren't!)

Reply:
OK. Sign, I wish everyone used braces.

Reply:
Floris and I decided to use braces like this for the school to keep parse errors better contained (otherwise the errors can get more confusing than they already are when you're first learning!)

Reply:
I understand the rationale! (or using `done` at the end of proofs, its just odd and potentially annoying for linters and stuff to have a mix of styles in mathlib)